src/wallet/wallet.cpp

```diff
@@ -2535,6 +2534,13 @@ bool CWallet::SelectCoins(const std::vector<COutput>& vAvailableCoins, const CAm
                                    vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)) {
                 return true;
             }
+            // Try with unsafe inputs if they are allowed. This may spend unconfirmed outputs
+            // received from other wallets.
+            if (coin_control.m_include_unsafe_inputs && SelectCoinsMinConf(value_to_select,
+                                                        CoinEligibilityFilter(0, 0, max_ancestors-1, max_descendants-1, true /* include_partial_groups */),
+                                                        vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)) {
```

Comment:
nit, I think it's best to not indent arguments the same as the function they are a part of. Maybe something like
```diff
-            if (coin_control.m_include_unsafe_inputs && SelectCoinsMinConf(value_to_select,
-                                                        CoinEligibilityFilter(0, 0, max_ancestors-1, max_descendants-1, true /* include_partial_groups */),
-                                                        vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)) {
+            if (coin_control.m_include_unsafe_inputs
+                && SelectCoinsMinConf(
+                    value_to_select,
+                    CoinEligibilityFilter(0, 0, max_ancestors-1, max_descendants-1, true /* include_partial_groups */),
+                    vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)) {
                 return true;
```

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/21359/commits/2bf0b1e7ee28dbc06aebf832c0480d257dad7294